CHARTER.md

```diff
@@ -0,0 +1,192 @@
+voting membership shares an employer, then the situation must be immediately remedied by the removal of voting member 
+status from one or more TSC voting members affiliated with the over-represented employer(s).
+
+The TSC shall meet regularly using tools that enable participation by the community (e.g. weekly on a Google Hangout On 
+Air, or through any other appropriate means selected by the TSC). The meeting shall be directed by the TSC Chairperson. 
+Responsibility for directing individual meetings may be delegated by the TSC Chairperson to any other TSC voting member. 
+Minutes or an appropriate recording shall be taken and made available to the community through accessible public 
+postings.
+
+TSC voting members are expected to regularly participate in TSC activities.
+
+A TSC voting member is automatically converted to a TSC regular member if they do not participate in three consecutive 
+TSC votes.
+
+## Section 4. Roles & Responsibilities of the TSC
+
+Subject to such policies as may be set by the CPC, the TSC voting members are responsible for all technical development 
+within the webpack project, including:
+
+* Setting release dates.
+* Release quality standards.
+* Technical direction.
+* Project governance and process.
+* GitHub repository hosting.
+* Conduct guidelines.
+* Maintaining the list of additional Collaborators.
+* Development process and any coding standards.
+* Mediating technical conflicts between Collaborators or Foundation projects.
+
+The TSC voting members will define webpack project's release vehicles.
+
+### Section 4.1. webpack Project Operations
+
+The TSC voting members will establish and maintain a development process for the webpack project. The development 
+process will establish guidelines for how the developers and community will operate. It will, for example, establish 
+appropriate timelines for TSC review (e.g. agenda items must be published at least a certain number of hours in advance 
+of a TSC meeting).
+
+The TSC and entire technical community will follow any processes as may be specified by the OpenJS Foundation Board 
+relating to the intake and license compliance review of contributions, including the OpenJS Foundation IP Policy.
+
+### Section 4.2. Decision-making, Voting, and/or Elections
+
+#### Section 4.2.1. Elections
+
+Leadership roles in the webpack project will be peer elected representatives of the community.
+
+For election of persons (such as the TSC Chairperson), a multiple-candidate method should be used, such as:
+
+* [Condorcet][] or
+* [Single Transferable Vote][]
+
+Multiple-candidate methods may be reduced to simple election by plurality when there are only two candidates for one 
+position to be filled. No election is required if there is only one candidate and no objections to the candidate's 
+election. Elections shall be done within the projects by the Collaborators active in the project.
```

Comment:
It looks like you're mixing up seats and candidates here. I suggest simplifying this section and copying it from what the CPC does for elections.

Reply:
@ovflowd can you find the relevant text we need to change to?

Reply:
Sure!